app/services/invoices/generate_pdf_service.rb

```diff
@@ -55,5 +57,9 @@ def should_send_webhook?
     def should_generate_pdf?
       context == 'admin' || invoice.file.blank?
     end
+
+    def charge?
+      invoice.fees.all? { |f| f.pay_in_advance? }
```

Comment:
Style/SymbolProc: Pass `&:pay_in_advance?` as an argument to `all?` instead of a block.